### PR TITLE
LDAP-318: AbstractContextSource.getUserDn() should be public

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

File: core/src/main/java/org/springframework/ldap/core/support/AbstractContextSource.java [hunk 1]
```diff
@@ -93,13 +93,13 @@
...
 	@Deprecated
-	protected String userDn = "";
+	private String userDn = "";
 
 	/**
 	 * @deprecated use {@link #getPassword()} and {@link #setPassword(String)} instead
 	 */
 	@Deprecated
-	protected String password = "";
+	private String password = "";
```
Comment (rwinch) [inline core/src/main/java/org/springframework/ldap/core/support/AbstractContextSource.java]: I think we need to leave this as protected so we don't break existing users.

File: core/src/main/java/org/springframework/ldap/core/support/AbstractContextSource.java [hunk 2]
```diff
@@ -93,13 +93,13 @@
 	 * @deprecated use {@link #getUserDn()} and {@link #setUserDn(String)} instead
 	 */
 	@Deprecated
-	protected String userDn = "";
+	private String userDn = "";
```
Comment (rwinch) [inline core/src/main/java/org/springframework/ldap/core/support/AbstractContextSource.java]: I think we need to leave this as protected so we don't break existing users. This is not a major release, so we should try to remain passive

**Comment by rwinch** [comment]
@eddumelendez Thanks for yet another PR! You are the Spring LDAP rock star :-D I have provided some comments inline.

**Comment by eddumelendez** [comment]
@rwinch Thanks for feedback. I will watching the project for the next major release too

**Comment by rwinch** [comment]
Thanks for the fast turnaround! This is now merged into master via 1ffd11a and this resolves gh-353

**Pull request closed without merging**